src/Loader/DoctrineDbalLoader.php

```diff
@@ -54,7 +55,14 @@ public function run(Queue $data): void
             $progressBar->start();
             while ($record = $data->pop()) {
                 if ($record instanceof QueryBuilder) { // Insert or Update Query
-                    $record->executeQuery();
+
+                    try {
+                        $record->executeQuery();
+                    } catch (Error $error) {
+                        $this->audit->auditException($error);
```

Comment:
Vai vajag šo audit() izsaukumu? Labi, ka Tu konvertē Error uz Exception, bet Pipeline runneris to vēlāk noķer un auditē automātiski. Šis būtu dubults ieraksts, šķiet.